docs/releasenotes/4.0.0.rst

```diff
@@ -16,6 +18,152 @@ or to install exactly this version::
 
     pip install robotframework-tidy==4.0.0
 
+New transformer RenameVariables (#354)
+---------------------------------------
+
+Transformer than renames and normalizes variable names.
+
+Following conventions are applied:
+
+- variable case depends on the variable scope (lowercase for local variables and uppercase for global variables)
```

Comment:
I think it should read "uppercase for non-local variables" or "uppercase for test, suite and global variables".

Reply:
I will go with ``uppercase for non-local variables``